osdep/w32_keyboard.c

```diff
@@ -43,7 +43,7 @@ static const struct keymap vk_map_ext[] = {
 static const struct keymap vk_map[] = {
     // special keys
     {VK_ESCAPE, MP_KEY_ESC}, {VK_BACK, MP_KEY_BS}, {VK_TAB, MP_KEY_TAB},
-    {VK_RETURN, MP_KEY_ENTER}, {VK_PAUSE, MP_KEY_PAUSE},
```

Comment:
I don't think you meant to remove `VK_PAUSE`?

Reply:
Why not? We have `APPCOMMAND_MEDIA_PAUSE` already.

Reply:
Oh I didn't see that. So that magically will work the same I guess?

Reply:
Ok, I tested. `APPCOMMAND_MEDIA_PAUSE` is for dedicated "media pause" button. And `VK_PAUSE` is `Pause/Break` key. Not the same thing. So if we want `Pause/Break` key to send `MP_KEY_PAUSE` we should restore it as you said.

Also I'm really confused at this.
```
#PLAY cycle pause                       # toggle pause/playback mode
#PAUSE cycle pause                      # toggle pause/playback mode
#PLAYPAUSE cycle pause                  # toggle pause/playback mode
#PLAYONLY set pause no                  # unpause
#PAUSEONLY set pause yes                # pause
```
```
#define MP_KEY_PLAY             (MP_KEY_MM_BASE+2)
#define MP_KEY_PAUSE            (MP_KEY_MM_BASE+3)
#define MP_KEY_PLAYPAUSE        (MP_KEY_MM_BASE+4)
#define MP_KEY_PLAYONLY         (MP_KEY_MM_BASE+23)
#define MP_KEY_PAUSEONLY        (MP_KEY_MM_BASE+24)
```

Ok, not related to this PR... I might clean this up. Pause key should pause, not cycle. So at least our mappings should use `MP_KEY_PAUSEONLY ` or do breaking change and remove all this cycle bs from PLAY and PAUSE key.

Reply:
Well I'm not sure if there's a reason to have `MP_KEY_PLAY` and `MP_KEY_PAUSE` in the first place. If we do keep them, then I don't see a problem with the current default mappings of cycle. But unrelated, yes.